test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -1081,32 +1081,20 @@ TEST(nntrainer_capi_nnmodel, get_input_output_dimension_03_n) {
   status = ml_train_model_construct_with_conf(s.getIniName().c_str(), &handle);
   EXPECT_EQ(status, ML_ERROR_NONE);
 
-  EXPECT_THROW(ml_train_model_get_input_tensors_info(handle, &input_info),
-               std::invalid_argument);
-  EXPECT_THROW(ml_train_model_get_output_tensors_info(handle, &output_info),
-               std::invalid_argument);
-
-  status = ml_tensors_info_destroy(input_info);
```

Comment:
I guess we need to add comment which shows that destroy is only required when the getting info is success.

Reply:
@zhoonit  does it has to be a separete PR to be included for ACR? or it can be done here?

Reply:
checked with @zhoonit offline, the comments for the destroy info will be added as a separate PR.